### PR TITLE
Reporting API documentation

### PR DESCRIPTION
**Pull request by Sonalit23**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -7060,3 +7180,361 @@ you'll see a response that isn't `200`.
...
+You can only download 31 days of data per request.
+
+#### Concurrent Requests
+Default: 3 requests per tenant
+
+This limits the number of concurrent downloads. The default is 3 concurrent downloads.
+If you exceed the quota, you'll get a 429 status code. The response will include the headers Retry-After, RateLimit-Limit, and RateLimit-Remaining.
+
+#### Download Volume
+Default: 1000 days of raw check data and 1000 days of raw issue data. The data will replenish at 5 days of data per day.
```
Comment (madeline-brennan) [inline apiary.apib:7206]:
```suggestion
Default: 1000 days of raw check data and 1000 days of raw issue data. The data will replenish at a rate of 5 days of data per day.
```

File: apiary.apib [hunk 2]
```diff
@@ -7060,3 +7180,361 @@ you'll see a response that isn't `200`.
...
+
+If you download any amount of data for a specific day, that day counts toward the default. This includes empty days. The 1000-day default is designed to give you access to all available historic data.
+
+The quota is set for each type of data. For example, when you download check data it won't affect the amount of issue data you can download.
+
+If you exceed the download quota, you'll get a 429 status code. The response will include the headers Retry-After, RateLimit-Limit, RateLimit-Remaining, and RateLimit-Reset headers.
```
Comment (madeline-brennan) [inline apiary.apib:7212]:
```suggestion
If you exceed the download quota, you'll get a 429 status code. The response will include the headers Retry-After, RateLimit-Limit, RateLimit-Remaining, and RateLimit-Reset.
```

File: apiary.apib [hunk 3]
```diff
@@ -7060,3 +7180,361 @@ you'll see a response that isn't `200`.
...
+
+**Note:** To access the Reporting API, you'll need to be authorized and authenticated via an access token. Before you get started with the Reporting API, be sure to review the [authentication and authorization](https://acrolinxapi.docs.apiary.io/#introduction/authentication-and-authorization) requirements.
```
Comment (madeline-brennan) [inline apiary.apib:7214]:
```suggestion
**Note:** To access the Reporting API, you'll need to be authorized and authenticated with an access token. Before you get started with the Reporting API, review the [authentication and authorization](https://acrolinxapi.docs.apiary.io/#introduction/authentication-and-authorization) requirements.
```

File: apiary.apib [hunk 4]
```diff
@@ -7060,3 +7180,361 @@ you'll see a response that isn't `200`.
...
+
+## Get check data [GET /api/v1/reporting/data/checks{?start,end}]
+
+Download check data as a CSV file. You can access the data with cURL. 
+
+**Note:** If no data is available, the CSV file will contain headers with zero rows of data. For example, data won't be available if no checks were run in the requested time range.
```
Comment (madeline-brennan) [inline apiary.apib:7220]:
```suggestion
**Note:** If no data is available, the CSV file will contain headers with zero rows of data. For example, data will only be available for checks run in the requested time range.
```

File: apiary.apib [hunk 5]
```diff
@@ -7060,3 +7180,361 @@ you'll see a response that isn't `200`.
...
+
+### Checks CSV data structure
+|Column name|Data type|Description|Length|
+|----------------|------------|------------|------------|
+|Check ID|string|Unique check ID|36|
+|Check start time|string|Start time of the check in the format `yyyy-MM-dd'T'HH:mm:ss.SSSZ`, for example, `2022-08-01T23:49:14.627Z`||
+|Language|string|Checking language|255|
+|Scorecard URI|string|Uniform Resource Identifier (URI) of the Scorecard|255|
+|File ID|string|Unique file ID|36|
+|Content reference|string|Acrolinx assigns the file a content reference the first time you run a check. Files with the same content reference are treated as the same file.|36|
```
Comment (madeline-brennan) [inline apiary.apib:7237]:
```suggestion
|Content reference|string|The first time you run a check, Acrolinx assigns the file a content reference. Acrolinx treats files with the same content reference as the same file.|36|
```

File: apiary.apib [hunk 6]
```diff
@@ -7060,3 +7180,361 @@ you'll see a response that isn't `200`.
...
+|Tone goal score|number|Score for the goal Tone||
+|{NAME} goal issues|number|Number of Issues found for the goal. The number of columns in the CSV file will correspond to the number of goal names. For example, if there are 10 goals, the CSV file will have 10 columns for goals.||
+|Clarity goal issues|number|Number of Issues found for the goal Clarity||
+|Consistency goal issues|number|Number of Issues found for the goal Consistency||
+|Correctness goal issues|number|Number of Issues found for the goal Correctness||
+|Inclusive Language goal issues|number|Number of Issues found for the goal Inclusive Language||
+|Scannability goal issues|number|Number of Issues found for the goal Scannability||
+|Terminology goal issues|number|Number of Issues found for the goal Terminology||
+|Tone goal issues|number|Number of issues found for the goal Tone||
+|{NAME} metric score|number|Metrics suffix by actual name. If there are 10 metrics, the CSV will have 10 columns for metrics suffix by name of the metric||
```
Comment (madeline-brennan) [inline apiary.apib:7273]:
```suggestion
|{NAME} metric score|number|Metrics suffix by actual name. If there are 10 metrics, the CSV will have 10 columns for metrics suffixes by name of the metric.||
```

**Review by madeline-brennan — APPROVED** [review]

**Comment by jeloba** [comment]
I'm unable to commit your suggestions @madeline-brennan, not sure why.

**Comment by madeline-brennan** [comment]
@jeloba maybe it has to do with the other unresolved stuff in the branch?

**Comment by jeloba** [comment]
@madeline-brennan I think you may need to make these changes directly to the `develop` branch, and not to this PR.

**Review by george-haddad — APPROVED** [review]

**Review by george-haddad — APPROVED** [review]

**Comment by rbarbey** [comment]
I've checked the preview and it looks good to me (wrt formatting etc)

**Pull request merged**